modules/social_features/social_landing_page/templates/page--node--landing-page.html.twig

```diff
@@ -27,6 +27,12 @@
  */
 #}
 
+{%
+  set classes = [
+  'row',
```

Comment:
@agami4 maybe it makes sense to add those classes together with `$variables['attributes']->addClass('container'`) in `social_landing_page.module` ?
Also if we will not remove those classes: `with-title-region', 'layout--with-complementary` it will not break any section of Landing Page?

Reply:
@ribel,
The `row` class adds to the current landing page.
The `container` class adds to the edit landing page

Reply:
And I've checked all paragraphs to the landing page and all looks good